packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -344,7 +371,7 @@ class AccountSwitcher extends React.Component {
                                             </Button>
                                         </div>
                                     ))}
-                                </React.Fragment>
+                                </>
```

Comment:
@zaki-binary doesn't like doing this, can you discuss and come up with what we want to follow?

Reply:
here he's changing all empty ones to fragments https://github.com/binary-com/deriv-app/pull/1099

Reply:
@negarn actually they were `<React.Fragment>` earlier! I changed them by @reza-fs 's suggestion. What is our convention for using `Fragment`?

Reply:
By the way, I changed all `<>`s to `<React.Fragment>` in this file